test/recipes/70-test_key_share.t

```diff
@@ -78,7 +78,7 @@ my $proxy = TLSProxy::Proxy->new(
 $testtype = EMPTY_EXTENSION;
 $direction = CLIENT_TO_SERVER;
 $proxy->filter(\&modify_key_shares_filter);
-if (disabled("ec")) {
+if (disabled("ec") || disabled("ecx")) {
```

Comment:
This does not make sense

Reply:
yes, but it failed after disabled ecx, any idea about it?
70-test_key_share.t ..
not ok 1 - Success after HRR
70-test_key_share.t .. 21/? ----------------------------------------------------